### DISCUSSION TITLE
not getting access of student developer pack even though i redeem it.

### DISCUSSION THREAD

**Discussion by ayeshaumraoshaik** [community/community#38905] [GitHub Education]
### Select Topic Area

Question

### Body

 i am not getting access of student developer pack even though i redeem it.
please answer my question why is it so?

**Comment by mdAshik922** [comment]
The correct information you provided for verification is not being
 approved by **Github Copilot** !! Or any other problem?
![image](https://user-images.githubusercontent.com/62794453/201450771-02ba649e-cc66-4eb2-ad6f-617a29180b76.png)

**Reply by ayeshaumraoshaik** [reply]
i have same problem .

**Reply by mdAshik922** [reply]
Is there such a thing in your profile?
![image](https://user-images.githubusercontent.com/62794453/201525141-fb3052aa-7dfa-420f-8dd9-abf6851c8b6c.png)

**Reply by mdAshik922** [reply]
Please read and follow this document

https://docs.github.com/en/copilot/quickstart

**Comment by dhrubasaha08** [comment]
Please contact Github Support !

**Comment by samus-aran** [comment]
Hi @ayeshaumraoshaik 👋 

Are you still having this issue? Let us know if it's been resolved or be explicit about what your problem entails (what benefits are missing?), giving a few more details might help someone give you a nudge in the right direction.

**Comment by x-dev-s** [comment]
I am having the same issue
